modules/system/console/MixCompile.php

```diff
@@ -95,40 +168,52 @@ protected function mixPackage($package)
             }
         });
 
-        $this->removeWebpackConfig($package['path']);
+        $this->removeWebpackConfig($mixJsPath);
 
         return $exitCode;
     }
 
-    protected function createCommand($package)
+    /**
+     * Create the command array to create a Process object with
+     */
+    protected function createCommand(string $mixJsPath): array
     {
+        $basePath = base_path();
         $command = $this->argument('webpackArgs') ?? [];
         array_unshift(
             $command,
-            $package['path'] . implode(DIRECTORY_SEPARATOR, ['', 'node_modules', 'webpack', 'bin', 'webpack.js']),
+            $basePath . '/node_modules/webpack/bin/webpack.js',
             '--progress',
-            '--config=' . $package['path'] . DIRECTORY_SEPARATOR . 'mix.webpack.js'
+            '--config=' . $this->getWebpackJsPath($mixJsPath)
```

Comment:
Does this work on Windows? While PHP might normalise paths, I'm not sure that the command line (or Process) does?

Reply:
I'd still want this to be tested - I distinctly remember Windows being a pain with pathing when I was writing the PathResolver class, so there's some contexts where Windows doesn't play nice with Unix paths.

Reply:
One of our devs runs windows so I'm fine with releasing and making him test it and then fixing it if it doesn't work 😂

Reply:
The path issues mostly come up when we are processing paths that have come from windows since they don't match Unix but sending paths to windows usually works fine